client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminPurchase.tsx

```diff
@@ -583,12 +660,30 @@ const AdminPurchase = () => {
             </button>
             <div className='mt-4 text-center'>
               <button
-                className='bg-green-600 px-7 py-2 text-sm font-medium text-white rounded-lg hover:bg-green-700 disabled:opacity-40 m-2'
-                onClick={handlePurchase}
+                className='bg-green-600 px-8 py-1 text-white rounded-xl hover:bg-green-700 disabled:opacity-40 m-2'
+                onClick={() => handlePurchase(false)}
               >
                 Proceed to Checkout
               </button>
             </div>
+            <div className="mt-4 text-center">
+              <h1>Select a Reader</h1>
```

Comment:
There should only be one `h1` per page otherwise it messes a little bit with screen readers, this could probably just be an `h2`

Reply:
reader select was made label